package.json

```diff
@@ -46,9 +46,9 @@
     "@radix-ui/react-navigation-menu": "^1.1.4",
     "@radix-ui/react-slot": "^1.0.2",
     "@radix-ui/react-tooltip": "^1.0.7",
-    "@sentry/nextjs": "^7.110.1",
-    "@sentry/opentelemetry-node": "^7.110.1",
```

Comment:
@sentry/opentelemetry-node is removed from v8 branch. No info about it in migration docs https://github.com/getsentry/sentry-javascript/blob/8.0.0-beta.1/MIGRATION.md

Reply:
Yeah, i was right to use `@sentry/opentelemetry` https://github.com/getsentry/sentry-javascript/pull/10906
Good to mention it in migration guide

Reply:
We will add it to the migration guide!